packages/app/src/systems/Account/services/account.ts

```diff
@@ -140,6 +144,14 @@ export class AccountService {
     }
   }
 
+  static async exportVault(input: AccountInputs['unlock']) {
+    const storage = new IndexedDBStorage() as never;
```

Comment:
Since you have this could also inside the `changePassword`, I think you could put it inside a function